spec/rubocop/cop/layout/line_length_spec.rb

```diff
@@ -261,6 +261,20 @@ def test_some_other_long_test_description_which_exceeds_length
       RUBY
     end
 
+    context 'and SplitStrings option is enabled' do
```

Comment:
There's already a context for correction when `SplitStrings` is set starting on line 529, can you please move this test into that block? You can add a context within it for heredocs.